docs/platforms/react-native/integrations/component-names.mdx

```diff
@@ -71,6 +71,10 @@ Here's what the resulting node would look like if your bundler had applied the p
 
 The Sentry browser SDK will pick off the value from these `data` attributes and collect them when your components are interacted with.
 
+## Troubleshooting
+
+- Use `export SENTRY_LOG_LEVEL=debug` to enable debug logs. When generating the JS Bundle, you should see logs about the Sentry Metro Plugin saving and loading the default Babel transformer.
```

Comment:
is there any more guidance we can give in case the logs don't show the expected lies?

Reply:
Thank you, yes, I've added more guidance.